lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -109,13 +108,6 @@ export default function NarrativeItinerariesHeader({
   const sortOptionsArr = sortOptions(intl, enabledSortModes)
   const sortText = sortOptionsArr.find((x) => x.value === sort.type)?.text
 
-  const handleSortClick = useCallback(
-    (value) => {
-      onSortChange(value)
-    },
-    [onSortChange]
-  )
-
```

Comment:
this useCallback is probably hurting performance more than anything. onSortChange won't be recreated on renders anyway since it's being passed in

Reply:
Love!